src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -40,6 +40,14 @@
       " damage to ALL enemies and lose all Coil."
     ]
   },
+  "stsjorbsmod:DoubleExhaustPower": {
+    "NAME": "Double",
+    "DESCRIPTIONS": [
+      "Your next card that #yExhausts is played twice this turn.",
+      "Your next ",
```

Comment:
In newer powers we've started preferring using `String.Format` and placeholder strings instead of splitting the string into parts like this and concatenating, since it's more localization-friendly. ie, instead of the last 2 DESCRIPTIONS entries here, `Your next %1$s cards that #yExhaust are played twice this turn.`